### ISSUE TITLE
在Houston任务上的精度比较差

### ISSUE THREAD

**Issue by Varro-pepsi** [pipi-jia/TGRS2024_S2ECNet#1]
可以发一下在Houston任务上具体的参数设置嘛，设置了与论文相同的参数训练后最终模型精度只有70%左右

**Comment by pipi-jia** [comment]
感谢您对工作的关注，我们刚刚使用"Namespace(batch_size=128, d_se=64, data_path='./datasets/Houston/', flip_augmentation=True, gpu=2, l2_decay=0.0001, log_interval=5, lr=0.001, lr_scheduler='none', max_epoch=400, mixture_augmentation=False, momentum=0.9, num_epoch=500, patch_size=13, pro_dim=256, radiation_augmentation=True, re_ratio=5, save_path='./results/final_check/', seed=233, source_name='Houston13', target_name='Houston18', test_stride=1, training_sample_ratio=0.8)"训练了40个epoch76.49的OA，方法中包含数据生成的部分，随机性较大，可能是种子的问题。

**Comment by Varro-pepsi** [comment]
感谢您的回信，我再去多训练几次

**Comment by abcd18383082610-web** [comment]
我有个问题啊，就是论文中应该是跑10次取平均，是随便写10个seed然后跑10次取平均吗

**Comment by Cerda-Link** [comment]
比如取随机种子1到10个跑一次然后取平均吗？

**Comment by pipi-jia** [comment]
感谢您对我们工作的关注，关于您的问题，可以按照您说的随便写几个种子，或者修改代码的相应部分把种子设置成random，然后多跑几次取均值就可以了。
